hedera/urls.py

```diff
@@ -22,6 +22,7 @@
     path("django-rq/", include("django_rq.urls")),
     path("account/login/", LoginView.as_view(form_class=LoginEmailForm), name="account_login"),
     path("account/signup/", views.SignupView.as_view(), name="account_signup"),
+    path("account/dashboard/", views.DashboardView.as_view(), name="dashboard"),
```

Comment:
I would also suggest just putting it as a top-level `/dashboard/` rather than under `/account/`. This is the pattern we've always followed with Pinax sites using django-user-accounts, etc.

Reply:
Just to clarify, can the dashboard view be accessible when the user is not logged in? I was working off the specs for issue #297, would definitely update the paths if the specs changed. let me know!

Reply:
I don't think the specs are changed. The user should be logged in to get to the dashboard, but that doesn't mean the url needs `account/` in it. To update my earlier suggestion, the path could be:
`path("dashboard/", login_required(views.DashboardView.as_view()), name="dashboard"),`
But you'd have to import `login_required` from the appropriate place.

Reply:
awesome thanks for the clarification @dodget !